### DISCUSSION TITLE
corenlp.run not working at the moment? 23.04.2024

### DISCUSSION THREAD

**Discussion by holdmysweater** [stanfordnlp/CoreNLP#1444] [General]
Hello! I'm so sorry, I just wasn't sure where to ask about this. 

I believe the website corenlp.run isn't working at the moment I am writing this. I was wondering what date it would be back on track (if it will be), because I'm using it for a project for literacy, would be very grateful to know!

Thank you for everyone working on this! Kind regards.

**Comment by AngledLuffa** [comment]
Kicked.  One of these days I need to instrument CoreNLP with enough
debugging information to figure out what queries are causing it to ESAD

**Reply by holdmysweater** [reply]
Thank you so much! You're a life-saver!

**Reply by holdmysweater** [reply]
So sorry to bother again, it's been down for 2 days again I think... ^-^

**Reply by AngledLuffa** [reply]
Please don't rely on us hosting the demo for anything.  Even when it's not the demo itself failing, our hardware often fails in one way or another and the demo becomes unreliable.  You can always host it yourself and it will entirely under your control whether or not it is working.

**Reply by holdmysweater** [reply]
My bad, I just don't know anything about this stuff. I'll ask around, thank you.
